lxd/db/cluster/schema.go

```diff
@@ -285,6 +285,23 @@ CREATE TABLE "networks" (
     type INTEGER NOT NULL DEFAULT 0,
     UNIQUE (project_id, name)
 );
+CREATE TABLE networks_acls (
+    id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+    project_id INTEGER NOT NULL,
```

Comment:
Shouldn't we have a foreign key to the project table?

Reply:
@stgraber yep I have added that now. The reason I didn't originally was that was was basing the table structure on the `networks` table, which doesn't currently have a foreign key on `project_id`. However expanding my search and looking at the `instances` table showed that there was a foreign key on there, so it looks like I missed this when adding project support for networks.